lncrawl/sources/wattpad.py

```diff
@@ -24,21 +24,22 @@ def read_novel_info(self):
         logger.debug('Visiting %s', self.novel_url)
         soup = self.get_soup(self.novel_url)
 
-        self.novel_title = soup.select('h1')[0].get_text().strip()
+        #self.novel_title = soup.select('h1')[0].get_text().strip()
+        self.novel_title = soup.select_one('.story-info__title').get_text().strip()
         logger.info('Novel title: %s', self.novel_title)
 
         self.novel_cover = self.absolute_url(
-            soup.select('div.cover.cover-lg img')[0]['src'])
+            soup.select_one('.story-cover img')['src'])
         logger.info('Novel cover: %s', self.novel_cover)
 
-        self.novel_author = soup.select(
-            'div.author-info strong a')[0].get_text()
+        self.novel_author = soup.select_one(
+            '.author-info__username').get_text()
         logger.info('Novel author: %s', self.novel_author)
 
-        #description = soup.select('h2.description')[0].get_text()
-
-        chapters = soup.select('ul.table-of-contents a')
-        # chapters.reverse()
+        len(soup.select_one('.story-parts').select('ul li a'))
+        
```

Comment:
Seems to do nothing, I assume you just missed it before submitting

Reply:
@yudilee please remove this line before I can merge it

Reply:
@Galunid @dipu-bd sorry, my bad forgot to clean up. I updated it . Thank you

Reply:
@yudilee Unless I'm missing something that `len(soup.select_one(...))` still does nothing and should be removed